translib/app_utils.go

```diff
@@ -183,14 +183,10 @@ func getTargetNodeYangSchema(targetUri string, deviceObj *ocbinds.Device) (*yang
 }
 
 func dumpIetfJson(s ygot.ValidatedGoStruct, skipValidation bool) ([]byte, error) {
-	jsonStr, err := ygot.EmitJSON(s, &ygot.EmitJSONConfig{
-		Format:         ygot.RFC7951,
-		SkipValidation: skipValidation,
-		RFC7951Config: &ygot.RFC7951JSONConfig{
-			AppendModuleName: true,
-		},
-	})
-	return []byte(jsonStr), err
+	cfg := ocbinds.EmitJSONOptions{
```

Comment:
The parameter "skipValidation" is not used?

Reply:
It is actually not in use -- callers of dumpIetfJson() always pass skipValidatuon=true.

Reply:
I removed the skipValidation parameter now